content/graphqlsearch/1.md

```diff
@@ -9,31 +9,34 @@ courseName = "Search"
 type = "graphql/schema"
 +++
 
-When Dgraph is searching for strings, dates, or other values based on a filter,
+When Dgraph searches for strings, dates, or other values based on a filter,
 it needs an index to make the search efficient. We've already seen examples of
 how to specify the indexes in a [schema update](../../graphqlschema/3).
 
-`Int`, `Float`, and `DateTime` have default indexes, but `String` has options of
-what index types to choose. Multiple indexes can be built for the same `String`
-valued predicate.
+`Int`, `Float`, and `DateTime` have default indexes, but `String` has a few
+options for which search index types to use. You can build multiple indexes
+for the same `String`-valued predicate.
 
-For `String` the following indexes are available
+For `String`, the following indexes are available:
 
-- `term` for use with filtering by all of terms and any of terms. Generates
-  filter inputs `allofterms` and `anyofterms`.
-- `exact` for use with inequality — matching whole string. Generates filter
-  inputs `lt`, `le`, `eq`, `ge`, `gt`, and `in`.
-- `hash` as for `exact` but hashes string - useful with long strings. Generates
-  filter inputs `eq` and `in`.
-- `fulltext` for full search engine like text search. Generates filter inputs
-  `alloftext` and `anyoftext`.
-- `regexp` for use with regular expressions searches. Generates filter input
-  `regexp`
+- `term`: Used when filtering by "all terms" or by "any terms"; generates filter
+   inputs `allofterms` and `anyofterms`
+- `exact`: for use with inequality, matches a whole string and generates the
+   following filter inputs: `lt` (<), `le` (<=), `eq` (=), `ge` (>=), `gt` (>),
+   and `in` (not equal)
+- `hash`: Used similarly to `exact`, but hashes the string to accommodate longer
+   strings; generates the following filter inputs: `eq` (equal) and `in` (not equal)
+- `fulltext`: Used for full-text search-engine style search; generates the following
+   filter inputs `alloftext` and `anyoftext`
+- `regexp`: Used with regular expression searches; generates the `regexp` filter
+   input
 
+<!-- TBD resolve this comment: Not sure what is meant by this sentence
```

Comment:
Referring to the syntax of using the directive in the schema. You put the indexes in a list. This can be understood from the examples without this sentence.

Reply:
I edited this to "You specify multiple indexes as a list passed to the `by` argument of the `@search` directive."